### ISSUE TITLE
Host parameter ignored

### ISSUE THREAD

**Issue by Magnum5234** [shirou/mqttcli#78]
Hey, thx for this tool :)

The host parameter is completely ignored.

```
% mqttcli pub -h 10.10.5.20 -t 'test' -m 'test message' -d
INFO[0000] Broker URI: tcp://localhost:1883             
INFO[0000] connecting...                                
INFO[0000] Topic: test                                  
INFO[0000] client connected                             
INFO[0000] Published                                    
INFO[0000] client disconnected                          
```

If fixed this by changing
https://github.com/shirou/mqttcli/blob/master/mqtt.go#L123

to
```go
host := c.String("h")
```

**Comment by shirou** [comment]
`h` is an alias where defined [here](https://github.com/shirou/mqttcli/blob/master/main.go#L117). So both `h` and `host` should work. Other alias parameters such as `p` and `port` are also works even if only `p` is used in [the code](https://github.com/shirou/mqttcli/blob/master/mqtt.go#L124). (Yes, I found inconsistency. but it works.)

**Comment by kevingarman** [comment]
My thanks as well, this is exactly what I was looking for!

The -h/--host parameter isn't working for me either....the config file works fine:

```
./mqttcli_linux_amd64 pub -d -h "192.168.101.25" -p "1883" -t "test/kg" -m "1"
INFO[0000] Broker URI: tcp://localhost:1883             
INFO[0000] connecting...                                
ERRO[0000] network Error : dial tcp 127.0.0.1:1883: connect: connection refused 

./mqttcli_linux_amd64 pub -d --host "192.168.101.25" -p "1883" -t "test/kg" -m "1"
Incorrect Usage: flag provided but not defined: -host
```

**Comment by shirou** [comment]
I noticed 0.1.0 release binaries do not include this `host` alias flag. I have released 0.2.0. Please try to use that version.

**Comment by kevingarman** [comment]
Yep, works for me....thanks!
